trio/_core/_run.py

```diff
@@ -475,7 +475,7 @@ def __enter__(self):
             task._activate_cancel_status(self._cancel_status)
         return self
 
-    def _close(self, exc, collapse=True):
+    def _close(self, exc, collapse=False):
```

Comment:
It doesn't look like this `collapse` parameter is used anymore, can you remove it?

Reply:
Done.